.pre-commit-config.yaml

```
@@ -0,0 +1,12 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+  - repo: https://github.com/ambv/black
+    rev: stable
+    hooks:
+    - id: black
+      language_version: python3.7
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.4.0
+    hooks:
+    - id: flake8
```

Ajoute une ligne vide à la fin du fichier
```suggestion
    - id: flake8

```

---

Est-ce que ça apporte une valeur supplémentaire d'avoir black ET flake8 ?

---

en fait ils sont complémentaires, **black** comme code formatter et **flake8** comme code checker ([source](https://ljvmiranda921.github.io/notebook/2018/06/21/precommits-using-black-and-flake8/))